common/nexus/endpoint_registry_test.go

```diff
@@ -380,6 +380,80 @@ func TestTableVersionErrorResetsPersistencePagination(t *testing.T) {
 	assert.Equal(t, int64(3), reg.tableVersion)
 }
 
+func TestRetryOnAbortedError(t *testing.T) {
```

Comment:
This is testing the client behavior, the code you changed is in the server.

Reply:
> Note for self: this is where the service does the retry portion.

That's not true, it's here:

https://github.com/temporalio/temporal/blob/ada5d0bbee45150888b63a132be8239fd5a87212/common/util.go#L302

See also the call site here:

https://github.com/temporalio/temporal/blob/ada5d0bbee45150888b63a132be8239fd5a87212/common/nexus/endpoint_registry.go#L281

Reply:
No need to test the client behavior in this PR.